src/python/pants/backend/python/tasks/pytest_run.py

```diff
@@ -475,7 +475,7 @@ def _do_run_tests_with_args(self, test_targets, pex, args):
 
       profile = self.get_options().profile
       if profile:
-        env['PEX_PROFILE_FILENAME'] = '{0}.subprocess.{1:.6f}'.format(profile, time.time())
+        env['PEX_PROFILE_FILENAME'] = '{}.subprocess.{:.6f}'.format(profile, time.time())
```

Comment:
this can still use an f-string! I believe
```suggestion
        env['PEX_PROFILE_FILENAME'] = f'{profile}.subprocess.{time.time():.6f}'.
```

maybe confirm in a REPL